docs/kubernetes/ingress-controller.md

```diff
@@ -18,6 +18,131 @@ features, that you can use to enhance your environment. For example
 ratelimits, circuitbreakers, blue-green deployments, shadow traffic
 and [more](ingress-usage.md).
 
+### Comparison with other Ingress controllers
+
+At Zalando we chose to run
+[kube-ingress-aws-controller](https://github.com/zalando-incubator/kube-ingress-aws-controller)
+with [skipper
+ingress](https://opensource.zalando.com/skipper/kubernetes/ingress-controller/)
+as target group. While AWS loadbalancer integration gives us features
+like TLS termination, automated certificate rotation, possible WAF,
+and security groups, the HTTP routing capabilities are very
+limited. Skipper's main advantage compared to other HTTP routers are
+matching and changing HTTP. Another advantage for us and for skipper
+users in general is that defaults with
+[kube-ingress-aws-controller](https://github.com/zalando-incubator/kube-ingress-aws-controller),
+just work as you would expect.
+
+There are a number of other ingress controllers including
+[traefik](https://traefik.io/),
+[nginx](https://kubernetes.github.io/ingress-nginx/),
+[haproxy](https://github.com/jcmoraisjr/haproxy-ingress) or
+[aws-alb-ingress-controller](https://github.com/kubernetes-sigs/aws-alb-ingress-controller). Why
+not one of these?
+
+HAproxy and Nginx are well understood and good TCP and HTTP proxies,
+that were built, before Kubernetes and their problem is on one hand
+the configuration files, that were build, when everything was quite
+static and their list of annotations to implement basic features are
+already quite a big list for users. When we tested both in a simple
+Mesos setup we saw problems that one of these instances returned HTTP
+500 for all requests, because of unknown reasons.  Other advantages to
```

Comment:
I would like to remove this: "When we tested both in a simple
Mesos setup we saw problems that one of these instances returned HTTP
500 for all requests, because of unknown reasons."

I believe it is not nice when open source project mentions in its docs a bug of another one.

Reply:
true